packages/codec/lib/types.ts

```diff
@@ -685,6 +685,16 @@ export interface CodeRequest {
   address: string;
 }
 
+/**
+ * A request for an ENS name
+ *
+ * @Category Requests
+ */
+export interface EnsRequest {
```

Comment:
I think this should include something about reverse resolution or "primary name" in the identifier here, just so it's more clear. Maybe `EnsPrimaryNameRequest`? I think I prefer that over `EnsReverseResolutionRequest`, but maybe go with whatever terminology you use elsewhere

Reply:
Sure, I could make that change.

Reply:
Thanks!